apps/etterlatte-trygdetid/src/test/kotlin/trygdetid/TrygdetidRepositoryTest.kt

```diff
@@ -0,0 +1,97 @@
+package no.nav.etterlatte.trygdetid
+
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import no.nav.etterlatte.libs.database.DataSourceBuilder
+import no.nav.etterlatte.libs.database.migrate
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.testcontainers.containers.PostgreSQLContainer
+import org.testcontainers.junit.jupiter.Container
+import trygdetid.beregnetTrygdetid
+import trygdetid.trygdetidGrunnlag
+import java.util.*
+import java.util.UUID.randomUUID
+import javax.sql.DataSource
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class TrygdetidRepositoryTest {
+
+    @Container
+    private val postgres = PostgreSQLContainer<Nothing>("postgres:14")
+    private lateinit var repository: TrygdetidRepository
+    private lateinit var dataSource: DataSource
+
+    @BeforeAll
+    fun beforeAll() {
+        postgres.start()
+        dataSource = DataSourceBuilder.createDataSource(postgres.jdbcUrl, postgres.username, postgres.password)
+        repository = TrygdetidRepository(dataSource.apply { migrate() })
+    }
+
+    @AfterAll
+    fun afterAll() {
+        postgres.stop()
+    }
+
+    @AfterEach
+    fun afterEach() {
+        cleanDatabase()
+    }
+
+    @Test
+    fun `skal opprette trygdetid`() {
+        val behandlingId = randomUUID()
+
+        val trygdetid = repository.opprettTrygdetid(behandlingId)
+
+        trygdetid shouldNotBe null
+        trygdetid.behandlingId shouldBe behandlingId
+    }
+
+    @Test
+    fun `skal opprette og hente trygdetid`() {
```

Comment:
Mangler test som sjekker at trygdetidgrunnlag er med i trygdetid. Kan legge det til i denne?

Reply:
Den testen der har ikke lagt ved noe trygdetidgrunnlag. Men i testen `skal opprette et trygdetidsgrunnlag` vil det returneres `Trygdetid` som inneholder `TrygdetidGrunnlag`. Alle bruker samme hent-funksjon for å returnere data så det testes i grunn der.